Source/WebCore/platform/graphics/freetype/FontPlatformDataFreeType.cpp

```diff
@@ -322,4 +322,41 @@ HbUniquePtr<hb_font_t> FontPlatformData::createOpenTypeMathHarfBuzzFont() const
 }
 #endif
 
+FontPlatformData FontPlatformData::create(const Attributes& data, const FontCustomPlatformData* custom)
+{
+    // Get some generic default settings from fontconfig for web fonts. Strategy
+    // from Behdad Esfahbod in https://code.google.com/p/chromium/issues/detail?id=173207#c35
+    // For web fonts, the hint style is overridden in FontCustomPlatformData::FontCustomPlatformData
+    // so Fontconfig will not affect the hint style, but it may disable hinting completely.
+    static std::once_flag flag;
+    static FcPattern* pattern = nullptr;
+    static bool fixedWidth = false;
```

Comment:
Move this to the else branch too

Reply:
I don't know if this is necessary. The  `pattern` and the `fixedWidth`  static definitions are used in L354. Because they are static definitions inside a function then the data is allocated statically, visible only inside the function  but, more important, only initialized the first time the block is entered.

Also if we let this as it is code right now we make more explicit that those variables will be always defined, no matters of what part of the if-then-else is chosen.

Reply:
It's not necessary, it just makes explicit that they are only used there and the comment only applies to the else branch too.

Reply:
this is true for the comment and also for the `flag` variable but still we need to define the fixedWidth and the pattern variables in the main block of the function to be available for the return statement (what uses them). I will update the patch.